src/style1st.css

```diff
@@ -0,0 +1,70 @@
+* {
```

Comment:
do you need this file?

Reply:
Please delete this file if you are not using it